junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -150,7 +151,7 @@
 	 * @since 5.3
 	 */
 	@API(status = EXPERIMENTAL, since = "5.3")
-	public static final String PARALLEL_CONFIG_CUSTOM_CLASS_PROPERTY_NAME = PARALLEL_CONFIG_PREFIX
+	public static final String PARALLEL_CONFIG_CUSTOM_CLASS_PROPERTY_NAME = JupiterTestEngine.PARALLEL_CONFIG_PREFIX
```

Comment:
We could leave the ones that reference `JupiterTestEngine` here. WDYT?

Reply:
Sure. What? ;-)